### PR TITLE
Docs on resource requirements for self-hosting

### PR DESCRIPTION
**Pull request by kaapstorm**
This documentation helps projects understand the resource requirements, and other considerations, for hosting CommCare.

Context: [SC-1818](https://dimagi-dev.atlassian.net/browse/SC-1818) and [SC-1844](https://dimagi-dev.atlassian.net/browse/SC-1844)

Note: This PR builds on Sravan's documentation. Its base branch is **sr/docs**. 

##### ENVIRONMENTS AFFECTED

None: Documentation only

### CODE REVIEW

File: hosting_docs/source/scaling/8-resource-requirements.rst [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+
+Single Server
+-------------
+
+This configuration is appropriate for projects with fewer than 1500
+users, each user with an active case load of fewer that 10,000 cases,
+and the project has no intention of growing.
+
+Specific hardware guidelines can become outdated dated quickly. The
```
Comment (Charl1996) [inline hosting_docs/source/scaling/8-resource-requirements.rst:99]: Should probably drop the second "dated"?

Comment (Charl1996) [inline hosting_docs/source/scaling/8-resource-requirements.rst:99]:
```suggestion
Specific hardware guidelines can become outdated quickly. The
```

**Review by proteusvacuum — APPROVED** [review]

File: docs/monitoring/setup_datadog.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.. _label_datadog-for-monitoring:
+
+Datadog for monitoring
+======================
+
+Datadog is a monotoring tool that has support for variety of applications and is easily extendable which is why in our case it is used for monitoring various system, application metrics and also custom CommCareHQ metrics. You can read more about datadog in their `docs <https://docs.datadoghq.com>`_.
+
+commcare-cloud can setup the requisite datadog integration automatically when you do full stack deploy. You will need to set ``DATADOG_ENABLED`` to ``True`` in your environment's public.yml file and add your account api keys to your vault file.
```
[the remark this answers is not in the document]
Comment (kaapstorm) [reply docs/monitoring/setup_datadog.rst]: 3bb8c51c2 -- incl. some blatant plagiarism.

File: hosting_docs/source/scaling/8-resource-requirements.rst [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+of server configurations that can host all the parts of the CommCare HQ
+software suite.
+
+When determining what configuration to use, it is important to keep the
+expected growth of the project in mind, in order to optimize costs and
+limit possible service interruptions due to server management
+operations. For example, if requisitioning more resources takes a while,
+it is better to have some buffer -- more drive space, RAM, or cores than
+necessary at first -- based on the expectation that by the time more
+resources come available, they will have become necessary.
```
[the remark this answers is not in the document]
Comment (kaapstorm) [reply hosting_docs/source/scaling/8-resource-requirements.rst]: Let me know whether f81562c91 clarifies the meaning here, or whether I should rethink how I explain this.

**Comment by kaapstorm** [comment]
It looks like I'm going to need to split commit c0e200d into one commit that renames `setup_datadog.md`, and another one that edits it, to resolve a conflict.

I'll wait until this is approved, and then I'll pull the **sr/docs** branch, and rebase, before merging.

**Comment by kaapstorm** [comment]
Picking up on [a comment][] by @sravfeyn , maybe I should move this document from the "scaling" section to the "installation prerequisites" section. It does look like it belongs under "[Team and hardware requirements](https://commcare-cloud.readthedocs.io/en/latest/installation/prereqs/4-requirements.html)".

  [a comment]: https://github.com/dimagi/commcare-cloud/pull/4963#issuecomment-960847963

**Comment by kaapstorm** [comment]
Closing in favor of PR #4990

**Pull request closed without merging**